openBCIBoard.js

```diff
@@ -1080,6 +1080,7 @@ function OpenBCIFactory() {
             while (readingPosition <= bytesToRead - k.OBCIPacketSize) {
                 if (data[readingPosition] === k.OBCIByteStart) {
                     var rawPacket = data.slice(readingPosition, readingPosition + k.OBCIPacketSize);
+                    this.emit('rawDataPacket',rawPacket);
```

Comment:
well you had originally asked for this with the EDF thing I believe, but this allows the user to interpret the data packet. It bypasses sample, so nothing interprets it.